packages/core/src/lib/TwaManifest.ts

```diff
@@ -289,6 +292,9 @@ export class TwaManifest {
       enableNotifications: DEFAULT_ENABLE_NOTIFICATIONS,
       shortcuts: shortcuts,
       webManifestUrl: webManifestUrl.toString(),
+      features: {locationDelegation: {
+        enabled: DEFAULT_ENABLE_LOCATION},
+      },
```

Comment:
nit: could you format this?

```
features: {
  locationDelegation: { enabled: DEFAULT_ENABLE_LOCATION },
}
```

(or having enabled on its own line is OK too)